File: UNIDAD-9/Cliente3EnRaya/src/domain/entities/Jugada.ts
//clase que representa una jugada para comunicarse con el servidor
export class Jugada {
    //array con las cordenadas del movimiento del usuario
    public movimiento: number[];
    //símbolo del jugador actual
    public simbolo: string;
    //constructor con el movimiento y el símbolo
    constructor(movimiento: number[], simbolo: string) {
        this.movimiento = movimiento;
        this.simbolo = simbolo;
    }
}

File: UNIDAD-9/Servidor3EnRaya/Servidor3EnRaya/Entities/Jugada.cs
namespace Servidor3EnRaya.Entities
{
    public class Jugada
    {
        //array que almacenará el movimiento de la jugada actual
        public int[] movimiento { get; set; }
        //almacena el símbolo de la jugada actual
        public string simbolo { get; set; }
        /// <summary>
        /// El objeto estará construido por el movimiento que haya hecho el
        /// jugador Y por el SÍMBOLO del jugador actual
        /// </summary>
        /// <param name="movimiento"></param>
        /// <param name="simbolo"></param>
        public Jugada(int[] movimiento, string simbolo)
        {
            this.movimiento = movimiento;
            this.simbolo = simbolo;
        }
    }
}